src/Uno.UI.Runtime.Skia.Gtk/Uno.UI.Runtime.Skia.Gtk.csproj

```diff
@@ -8,8 +8,6 @@
 		<TargetFrameworks>$(TargetFrameworks);net7.0</TargetFrameworks>
 	</PropertyGroup>
 
-	<Import Project="../netcore-build.props" />
```

Comment:
It looks like this import can re-add `net7.0` which was already added in the previous `PropertyGroup`.